composer.json

```diff
@@ -18,7 +18,7 @@
         "ext-tokenizer": "*",
         "symfony/console": "^2.3 || ^3.0",
         "symfony/event-dispatcher": "^2.1 || ^3.0",
-        "symfony/filesystem": "^2.1 || ^3.0",
+        "symfony/filesystem": "^2.4 || ^3.0",
```

Comment:
if you drop Symfony 2.3, please bump to 2.7 directly. It does not make sense to use 2.4 here IMO, as it is unmaintained since years